script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1391,109 +1586,133 @@ I don't give a damn whether she was being serious or just trying to rile me up.
 [sha:d01136379096ad8f457455cf8bf12aac76236456]{
 -- Page 252, Offset 22786.
 -- 先輩にとってアルクェイドは倒すだけの吸血鬼で、そんなものを信じる俺は愚か者だと思っている。
-To Senpai, Arcueid is nothing more than a vampire that must be defeated, and she thinks I'm a fool for trusting her.
+It's clear that either way,
+ Senpai thinks Arcueid is nothing more than a vampire to be defeated,
+ and that I'm a fool for trusting her.
 }
 [sha:49995e4ab7f9806145ef55e1a2452aac7ad17d18]{
 -- Page 253, Offset 22787.
 -- 「俺は騙されてなんかないっ！　だいたい、アイツが嘘なんかつけるもんか、先輩みたいに器用じゃないんだから！
-"I'm not being deceived! Arcueid doesn't lie anyway, she isn't cunning like you!
+// 2 dumb 2 lie
+"I'm not being deceived!
+ As if she's even capable of lying in the first place.
+ She's not a snake like you, Senpai!
 }
 [sha:42c436afdaf915d0481278b6534ff4e9d1aa27c5]{
 -- Page 253, Offset 22788.
 -- 俺はアイツを信じます。っていうか、アレが嘘でも喜んで騙されてやる！」
 // Ciel big L in her own route
-I trust her. Scrap that, even if she's lying, I'll gladly be deceived!"
+I believe what she said.
+ Fuck it, even if she %{i}is%{/i} lying, I'll gladly be fooled by her!"
 }
 [sha:5ad664558d4e40872b8debd606e43987ac1043bf]{
 -- Page 254, Offset 22789.
 -- 「あのですね。自分が何を言っているのか理解していますか？　社会を守る私たちより、自由気ままに動いては街を混乱に陥れる、あのはた迷惑な真祖を信じるって言ったんですよ？
-"Excuse me? Do you even understand what you're saying? Did you just tell me that you'd believe that irksome True Ancestor who plunged this city into chaos on a whim over people like us who are protecting society?
+"Excuse me?
+ Can you even hear yourself right now?
+ She plunged this city into chaos on a whim.
+ Did you seriously just tell me that you believe that devil-may-care True Ancestor over us,
+ the people who are actually working to %{i}protect%{/i} society?
```

Comment:
自由気ままに動いては街を混乱に陥れる is actually quite a broad statement.
>Excuse me?
> Can you even hear yourself right now?
> Her going around and doing as she pleases is enough to plunge the city into chaos!
> Did you seriously just tell me that you believe that pain-in-the-ass True Ancestor over us,
> the people who are actually working to %{i}protect%{/i} society?

But perhaps something more fundamental with the structure can be changed to accommodate this change. It's more about what Ciel _believes_ Arcueid is, rather than what she's done.
Also, devil-may-cry might as well be pain-in-the-ass, because 迷惑な is more in that vein.